apps/passport-server/src/util/devconnectTicket.ts

```diff
@@ -28,11 +30,29 @@ export function pretixTicketsDifferent(
     return true;
   }
 
+
   if (
     oldTicket.pretix_checkin_timestamp !== newTicket.pretix_checkin_timestamp
   ) {
-    return true;
+    // The inequality might be because one of these is null, but it might
+    // also be because both are date objects, in which case we need a
+    // stricter check.
+    if (oldTicket.pretix_checkin_timestamp instanceof Date &&
+      newTicket.pretix_checkin_timestamp instanceof Date) {
+        if (oldTicket.pretix_checkin_timestamp.getTime() !== newTicket.pretix_checkin_timestamp.getTime()) {
+          return true;
+        }
+    }
+    else {
```

Comment:
i don't think you've run the autoformatter on this file

Reply:
`prettier` seems to be broken for me in VSCode. I'll see if I can figure out what's wrong with it.

Reply:
sometimes it's necessary to restart vscode to fix that. super annoying. https://github.com/proofcarryingdata/zupass/issues/718